salt/modules/disk.py

```diff
@@ -422,6 +422,7 @@ def format_(device,
             fs_type='ext4',
             inode_size=None,
             lazy_itable_init=None,
+            fat=None,
```

Comment:
do we really want this new parameter? What about using fs_type for fat, as in fat12, 16, 32?

Reply:
We do need it, as mkfs is a frontend for the specific executable mkfs.fs_type. There is not mkfs.fat12, mkfs.fat16 nor mkfs.fat32 executables, only mkfs.fat of mkfs.vfat. They use the -F parameter to indicate the size.

Reply:
I think it's a wrong route to take, adding each different specific mkfs parameters here. The point of this function is to unify those calls, so probably we should have an abstraction parameter for this, where on ext to mean inode_size and on fat/vfat to mean fat-size. I'm not saying to reuse this particular inode_size, but to avoid adding new parameters for very specific calls.

At worst, we should call it fat-size, as their man page describe it - https://linux.die.net/man/8/mkfs.vfat